demisto_sdk/commands/common/tests/dependencies_test.py

```diff
@@ -256,7 +258,7 @@ def create_inputs_for_method(repo, current_pack, inputs_arguments):
                      get_entity_by_pack_number_and_entity_type(repo, current_pack, inputs_arguments[0])}
 
     inputs_arguments = inputs_arguments[1:]
-    number_of_packs = len(repo.packs)
+    number_of_packs = len(repo.packs) - 1
```

Comment:
why?

Reply:
Since I add the `CommonTypes` pack, and I dont want any of its entities to be included in the regular dependency flow I used this line to ignore it.
I will add a comment